src/cmap/auth/gssapi.ts

```diff
@@ -1,9 +1,30 @@
 import { AuthProvider, AuthContext } from './auth_provider';
 import type { Callback } from '../../types';
 import { MongoError } from '../../error';
-
 import { Kerberos } from '../../deps';
 
+import * as dns from 'dns';
+
+interface KerberosClient {
```

Comment:
I think this might better belong in the `deps` file next to the `Kerberos` definition. Eventually the types should be moved to the `kerberos` library completely